roles/django_app_k8s/templates/rabbitmq-svc.yml.j2

```diff
@@ -0,0 +1,21 @@
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app.kubernetes.io/name: rabbitmq
+    app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+    app.kubernetes.io/version: "3.7"
```

Comment:
I'd correlate that to the rabbitmq image tag - I see that latest is on 3.10 by now

Reply:
added and using: `django_app_k8s_rabbitmq_version: "3.10"`